### DISCUSSION TITLE
100balancebms not work

### DISCUSSION THREAD

**Discussion by DO5MZ** [mr-manuel/venus-os_dbus-serialbattery#140] [Q&A]
Hy 
my 100balancebms  does not work. I use the original 100 Balance USB 485 adapter an i is it on the venus OS with LSUSB.
I use venus OS 3.52 and the latest release  of seriealbattery.
What am I doing wrong? Does anyone have a tip for me?

by Markus

**Comment by mr-manuel** [comment]
You checked the [supported BMS](https://mr-manuel.github.io/venus-os_dbus-serialbattery_docs/general/supported-bms), right?

**Comment by DO5MZ** [comment]
100balancebms is a Daly BMS with 1A balance current

**Reply by transistorgit** [reply]
I have 2 100balance BMS here and they work like normal Dalys

Check your wiring and your config. 

Try to attach the USB converter to a PC and use the PC software to read out the bms to check if its working

https://www.b100bms.com/download.asp
